resources/definitions/ubuild-3d_mr_bot_280.def.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "ubuild-3d_mr_bot_280",
+    "version": 2,
+    "name": "uBuild-3D Mr Bot 280",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "uBuild-3D",
+        "manufacturer": "uBuild-3D",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "icon": "icon_uBuild-3D",
+        "platform": "mr_bot_280_platform.stl",
+        "has_materials": true
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Mr Bot 280" },
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 275 },
+        "machine_height": { "default_value": 275 },
+        "machine_depth": { "default_value": 275 },
+        "machine_center_is_zero": { "default_value": false },
```

Comment:
The default value for this is false, so it doesn't have to be added here.

Reply:
I'll fix it asap